<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassificationWidgetClass</class>
 <widget class="QWidget" name="ClassificationWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClassificationWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="trainingGroupBox">
     <property name="title">
      <string>Training</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="methodComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>SVM</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Method: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="svmTrainingParsGroupBox">
        <property name="title">
         <string>Training Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Kernel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="kernelComboBox">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Linear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Polynomial</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RBF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sigmoid</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Degree</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="degreeDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Coef0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="coef0DoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Gamma</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="gammaDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="5">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Cost Parameter C</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2" colspan="2">
          <widget class="QDoubleSpinBox" name="CDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
           <property name="value">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="5">
          <widget class="QGroupBox" name="crossValidationGroupBox">
           <property name="toolTip">
            <string>Parameters for finding optimal parameters using cross validation</string>
           </property>
           <property name="title">
            <string>Use cross validation to find optimal parameters</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QSpinBox" name="nFoldSpinBox">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string>Fold = </string>
              </property>
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Degree grid (min, max, log-step) = </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="degreeMinSpinBox">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="degreeMaxSpinBox">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="degreeStepSpinBox">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Coef0 grid (min, max, log-step) = </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="coef0MinSpinBox">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="coef0MaxSpinBox">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="coef0StepSpinBox">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Gamma grid (min, max, log-step) = </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="gammaMinSpinBox">
                <property name="decimals">
                 <number>6</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="gammaMaxSpinBox">
                <property name="decimals">
                 <number>6</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="gammaStepSpinBox">
                <property name="decimals">
                 <number>6</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>C grid (min, max, log-step) = </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="CMinSpinBox">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="CMaxSpinBox">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="CStepSpinBox">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="setDefaultGridsPushButton">
              <property name="text">
               <string>Set Default Grids</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="trainPushButton">
          <property name="text">
           <string>Train</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="loadTrainingSetFromFilePushButton">
          <property name="text">
           <string>Load training set from XML file</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="saveTrainingSetFromFilePushButton">
          <property name="text">
           <string>Save training set to XML file</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="loadedTrainingSetInfoLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>info on training set</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="crossValidationInfoLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>label</zorder>
     <zorder>methodComboBox</zorder>
     <zorder>svmTrainingParsGroupBox</zorder>
     <zorder>gridLayoutWidget</zorder>
     <zorder>setDefaultGridsPushButton</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="classificationGroupBox">
     <property name="title">
      <string>Classification</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
